src/textord/tablefind.cpp

```diff
@@ -1978,8 +2008,30 @@ void TableFinder::MakeTableBlocks(ColPartitionGrid *grid, ColPartitionSet **all_
       table_partition->set_flow(BTFT_CHAIN);
       table_partition->SetBlobTypes();
       grid->InsertBBox(true, true, table_partition);
+      
```

Comment:
The following inserted block causes the assertion. Disabling that block fixes the regression.